main.py

```diff
@@ -1,12 +1,197 @@
+            response = video_store.edit_video(video_id=video_id, title=title, release_date=release_date, total_inventory=total_inventory)
+
+            print_stars()
+            print("Updated video: ")
+            print(response)
+            print_stars()
+
+        elif choice == '5':
+            print("Which video would you like to delete?")
+            video_id = input("Enter a video id: ")
+            video_store.delete_video(video_id)
+            print_stars()
+            print("Video has been deleted.")
+            print_stars()
+
+        elif choice == '6':
+            print_stars()
+            for customer in video_store.get_customers_info():
+                # print(customer)
+                print(f"ID: {customer['id']} - {customer['name']} - Phone: {customer['phone']} - Rented Videos: {customer['videos_checked_out_count']}")
+            print_stars()
+
+        elif choice == '7':
+            print("Wonderful! Let's add a new customer!")
+            name=input("What is the name of the customer? ")
+            postal_code=input("What is the postal code of the customer? ")
+            phone=input("What is the phone number of the customer? ")
+            response = video_store.add_customer(name=name, postal_code=postal_code, phone=phone)
+            print_stars()
+            print("New customer: ") 
+            print(response)
+            print_stars()
+
+        elif choice == '8':
+            print("Whohoo! Let's choose a customer!")
+            customer_id = input("Enter a customer id: ")
+            response = video_store.get_customer_info(customer_id)
+            print_customer(response)
+
+        elif choice == '9':
+            print("Wonderous! Let's update the customer!")
+            customer_id = input("Enter a customer id: ")
+            name=input("What is the name of the customer? ")
+            postal_code=input("What is the postal code of the customer? ")
+            phone=input("What is the phone number of the customer? ")
+            response = video_store.edit_customer(customer_id=customer_id, name=name, postal_code=postal_code, phone=phone)
+            print_stars()
+            print("Updated customer: ")
+            print(response)
+            print_stars()
+
+        elif choice == '10':
+            print("Which customer would you like to delete?")
+            customer_id = input("Enter a customer id: ")
+            video_store.delete_customer(customer_id)
+            print_stars()
+            print("The customer has been deleted.")
+            print_stars()
+
+        elif choice == '11':
+            print("What video would you like to check out? ")
+            customer_id = input("Enter the customer id: ")
+            video_id = input("Enter the video id: ")
+            response = video_store.check_out_video(customer_id=customer_id, video_id=video_id)
+            print_stars()
+            print("The video was checked out! ")
+            print(response)
```

Comment:
Consider how to handle a video that has not available inventory. How could our CLI handle a 400 or 404 level response?